go/pkg/metastore/db/dao/collection_test.go

```diff
@@ -97,11 +97,19 @@ func (suite *CollectionDbTestSuite) TestCollectionDb_GetCollections() {
 	collections, err = suite.collectionDb.GetCollections(nil, nil, suite.tenantName, suite.databaseName, &limit, nil)
 	suite.NoError(err)
 	suite.Len(collections, 1)
-	suite.Equal(collectionID, collections[0].Collection.ID)
+	if collectionID < collection2 {
```

Comment:
Would it be cleaner to check for either fo the IDs like so:

```go
suite.Contains([]string{collectionID, collection2}, collections[0].Collection.ID)
```

Reply:
The previous test was checking for the correct order and I kept that logic. The question is - do we want to test the order of how collections are returned. I decided that we better test it...
I don't have a strong opinion. I guess the order is not that important. I am fine not testing and making the code slightly simpler.
Fine either way - would keep it as it is if up to me.

Reply:
The test assumes that the second ID added will show as the latest, but as you point out with UUIDs, that is not true, and therefore, the whole ID check is relatively futile. UUIDs don't work with lexicographic sorting. A simple test like this demonstrates it:

```go
import (
	"fmt"
	"github.com/google/uuid"
	"sort"
	"testing"
)

func TestCreateTenant(t *testing.T) {
	uuidWithHyphen0 := uuid.New()
	uuidWithHyphen1 := uuid.New()
	lst := []string{uuidWithHyphen0.String(), uuidWithHyphen1.String()}
	fmt.Println(lst)
	sort.Strings(lst)
	fmt.Println(lst)
}
```

So we can either drop the ID check altogether or check for either of the IDs

Reply:
Currently, the test assumes that the collections will be returned ordered by id. That's why it compares the first collection returned with the smaller collectionID and the second with the larger collectionID.
IDs are random but `GetCollections` with offset depends on the stable id order. I would actually keep the order checks because if that is broken - offset would not work. Offset needs a stable order.

OK with keeping the order check?

Reply:
Or maybe it is even better to get the lexicographically sorted Ids prior to making the ID checks. This way we know how UUIDs of collections are sorted and should appear with/without offset and limit.

Reply:
Yeah - that way will be even cleaner. Will do it. Thank @tazarov.